Client/WebFilters/AdblockResourceDownloader.swift

```diff
@@ -34,9 +34,22 @@ class AdblockResourceDownloader {
         Preferences.Shields.useRegionAdBlock.observe(from: self)
     }
     
+    /// Initialized with year 1970 to force adblock fetch at first launch.
+    private(set) var lastFetchDate = Date(timeIntervalSince1970: 0)
+    
     func startLoading() {
-        AdblockResourceDownloader.shared.regionalAdblockResourcesSetup()
-        AdblockResourceDownloader.shared.generalAdblockResourcesSetup()
+        let now = Date()
+        let fetchInterval = AppConstants.buildChannel.isPublic ? 6.hours : 10.minutes
```

Comment:
blocked to decide on fetch interval, i like to fetch more often than 24hours, set it to 6 for now

Reply:
can we add some randomization to this so it's not as much of a user fingerprint? ex: each time the fetch is supposed to occur, delay it by a random amount between 1 and 10 minutes.

Reply:
@diracdeltas This will only be non-random if the user happens to open their phone every 6 hours since it only triggered on app foreground, not on a specific timer.